src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -82,67 +84,84 @@ const ProjectInstallPlatform = React.createClass({
     );
   },
 
-  render() {
+  renderSidebar() {
+    let platform = this.state.platform;
+    return (
+      <div className="install-sidebar col-md-2">
+        {this.props.platformData.platforms.map((p_item) => {
+          return (
+            <LanguageNav name={p_item.name} active={platform && platform.id === p_item.id}>
+              {p_item.integrations.map((i_item) => {
+                return this.getPlatformLink(i_item.id, (i_item.id === p_item.id ? t('Generic') : i_item.name));
+              })}
+            </LanguageNav>
+          );
+        })}
+      </div>
+    );
+  },
+
+  renderBody() {
     let {integration, platform} = this.state;
     let queryParams = this.props.location.query;
     let {orgId, projectId} = this.props.params;
 
+    if (!(integration && platform)) {
```

Comment:
Maybe this should be: `if (!integration || !platform) {`

Basically, if one of integration or platform are not present, bail out.